src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -678,6 +678,12 @@ private static object AddPsProperties(object psObj, object obj, int depth, bool
         /// <param name="context">The context for the operation.</param>
         private static void AppendPsProperties(PSObject psObj, IDictionary receiver, int depth, bool isCustomObject, in ConvertToJsonContext context)
         {
+            // if the psObj is a DateTime or String type, we don't serialize any extended or adapted properties
+            if (psObj.BaseObject is DateTime || psObj.BaseObject is string)
```

Comment:
It seems strings are more frequent:
```suggestion
            if (psObj.BaseObject is string || psObj.BaseObject is DateTime)
```